docs/ui/insight/get-started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Get started with RedisInsight
+linkTitle: Get started
+description: Get started with RedisInsight
+weight: 20
+
+---
+
+RedisInsight is available as a standalone download or [bundled with Redis Stack](/docs/stack/get-started/install). 
+Standalone downloads are available for Windows, MacOS, and Linux from the Redis Enterprise website and stores.
+
+## Install on Windows
+
+**From website**
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Open the `RedisInsight-v2-win-installer.exe` file.
+
+1. Open the RedisInsight icon.
+
+**From stores**
+
+* Go to [Microsoft Windows Apps](https://apps.microsoft.com/store/detail/redisinsight/XP8K1GHCB0F1R2) and download the RedisInsight app.
+
+## Install on MacOS
+
+**From website**
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Add RedisInsight to **Applications**. 
+
+**Using Homebrew**
+
+```
+brew install --cask redisinsight
+```
+
+To start RedisInsight, from your Terminal, run this command.
+
+```
+redisinsight
+``` 
+
+**From stores**
+
+* Go to [Apple Store](https://apps.apple.com/us/app/redisinsight/id6446987963) and download the RedisInsight app.
+
+
+## Install on Linux
+
+**From website**
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Open the Linux command line and install the package. 
+
+   * For Debian, run:
```

Comment:
let's display the AppImage option first because it is for any common Linux distributions (including Debian), and we can say it like this:
* For the`.AppImage` package:
   - To make your downloaded file...
* For the`.deb` package, run:

because both options may work for Debian